hubspot/management/commands/sync_hubspot.py

```diff
@@ -123,6 +140,7 @@ def sync_all(self):
         self.sync_line_items()
         self.sync_b2b_deals()
         self.sync_b2b_product_item()
+        self.sync_b2b_contacts()
```

Comment:
Reorder these so that b2b contacts and products are synced before deals:
```python
self.sync_b2b_contacts()
self.sync_b2b_product_item()
self.sync_b2b_deals()
```

Reply:
Actually I think products should remain after deals, but contacts should go first, sorry:
```python
self.sync_b2b_contacts()
self.sync_b2b_deals()
self.sync_b2b_product_item()
```